.remarkrc.cjs

```diff
@@ -0,0 +1 @@
+module.exports.plugins = [require("remark-preset-lint-origami-component")]
```

Comment:
This is a great addition 🙌 
I was working on a component yesterday that had linting errors in the readme and I was hoping my editor would highlight the issues, if the component had this config file then my editor would have done that. Instead I had to keep running `obt verify` and checking the output manually.

Reply:
yes:D i think @notlee did this